### DISCUSSION TITLE
Want to calculate daily climatologies from land restarts for initialized prediction

### DISCUSSION THREAD

**Discussion by juliecaron** [ESCOMP/CTSM#1831] [General]
hi all, i want to take existing restart files from a previous run (CESM2, FV 1 deg), and calculate climatologies for each day to use to initialized climatological land for prediction runs. i'm not terribly familiar with land restart files and there are a significant number of variables on the files (clm2.r.). i'd like help to do this correctly. i'd like to know which variables cannot be averaged because they are dimensions, don't change from one time to the next, or for other reasons (like the average would create a wrong answer). i'd be happy to set up a zoom call with relevant knowledgable land folks on how to do this and we are always happy to include collaborators on the project and resulting paper. thank you!

**Comment by ekluzek** [comment]
@juliecaron we had some offline discussion on this. Did you get this workin? Is this still an outstanding thing to work on?

**Comment by juliecaron** [comment]
hey erik,
this work was handed off to sasha glanville, i believe. maybe check with
her?
thanks!
julie
